mpmp/src/cmds/Subscribe.java

```diff
@@ -25,12 +25,7 @@ public void exec(String line, Conn conn) {
 			return;
 		}
 
-		for (String s : Arrays.copyOfRange(args, 3, args.length)) {
-			if (name == null)
-				name = s;
-			else
-				name = name + " " + s;
-		}
+		name = String.join(" ", Arrays.copyOfRange(args, 3, args.length));
```

Comment:
I had this feeling that there *must* be such a function is Java, but before today I didn't bother looking it up. Before today, mind you, I never had to write one of these stupid loops, it was just you. That's why I didn't change it. ^^